client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningPersonal.tsx

```diff
@@ -18,6 +18,8 @@ import IdentificationTextField from 'commons/IdentificationTextField/Identificat
 
 import useStyles from './ContactQuestioningStyles';
 import { ADDITIONAL_PHONE_LABEL } from '../PersonalInfoTab/PersonalInfoTab';
+import GroupedInteractedContact from 'models/ContactQuestioning/GroupedInteractedContact';
+import formatDate from '../../../../../../Utils/DateUtils/formatDate';
```

Comment:
use path of '/Utils/DateUtils/formatDate'

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1384/commits/eab3efc89f4a2466c96ab9b7ad6b816da877fe97)